src/AggregateFunctions/AggregateFunctionSparkbar.h

```diff
@@ -319,7 +327,7 @@ class AggregateFunctionSparkbar final
     {
         auto & to_column = assert_cast<ColumnString &>(to);
         const auto & data = this->data(place);
-        const String & value = render(data);
+        const auto & value = render(data);
```

Comment:
Let's make the `render` function insert directly into the column, and avoid any copying, and then even a `using` directive will be unneeded.